package.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "@amplitude/react-native": "2.7.0",
     "@atlaskit/button": "15.1.4",
+    "@atlaskit/checkbox": "12.0.0",
```

Comment:
This looks like a merge error, it's now gone.

Reply:
Fixed